### DISCUSSION TITLE
我的动态页面，左边直播栏没有滚轮无法滚动预览全部关注直播间

### DISCUSSION THREAD

**Discussion by wcnpdcwxy** [the1812/Bilibili-Evolved#3870] [综合讨论 / General]
各位大佬，我的动态页面，左边直播栏没有滚轮无法滚动预览全部关注直播间，是bug还是哪里设置的不对啊？

**Comment by the1812** [comment]
#3774?

**Reply by wcnpdcwxy** [reply]
感谢，非常类似这个问题，我这边是能显示10个直播间信息，而且根据我使用经验，有一阵子可以正常使用滚轮滚动查看，有一阵子又不行了，原因未知。
